### PR TITLE
Polishing up Kneel Diamond.

### PR DESCRIPTION
**Pull request by heymonicakay**
## Changes Made
- Reworded some of the explanations, and emphasized some of the important terms in chapters 4, 6, and 8 of Kneeling Diamond.
- Removed setMetal function from the import, and updated the variable names in the .map() portion of the example code provided in chapter 5.
- Added a link to dbdiagram in ch apter 3.
- Updated variable names in "DiamondSizes.js" in the boilerplate code script from "metals" to "sizes".

### CODE REVIEW

File: book-4-kneel-diamonds/chapters/KD_RADIO_BUTTONS.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Showing Custom Jewelry Options
 
-If you hadn't noticed yet, take a look in the `main.js` module. You'll notice that, unlike previous projects, it doesn't define the main HTML structure of your application. It is deferring that responsibility to the **`KneelDiamonds`** component.
+If you hadn't noticed yet, the `main.js` module looks a bit different than it has in previous projects. It doesn't define the main HTML structure of your application. It is deferring _that_ responsibility to the **`KneelDiamonds`** component.
 
-There is a function named `renderAllHTML()` that is then immediately invoked. This function is needed later in the growth of the application, because for Kneel Diamonds, you are going to react to the user choosing options.
+There is a function named `renderAllHTML()` that is definted in the `main.js` module, then immediately invoked. This function is needed later in the growth of the application, because for Kneel Diamonds, you are going to react to the user choosing options.
```
Comment (stevebrownlee) [inline book-4-kneel-diamonds/chapters/KD_RADIO_BUTTONS.md:5]: defined

File: book-4-kneel-diamonds/chapters/KD_RADIO_BUTTONS.md [hunk 2]
```diff
@@ -16,30 +16,30 @@ Give it a shot, and if you need to, you can [peek at the solution](./images/knee
...
+Once you show metals, also render the components that show the options for diamond sizes and jewelry styles.
 
 ## The Map Array Method
 
 | | |
 |:---:|:---|
 | <h1>&#x270e;</h1> |  _The **map()** array method is a conversion tool. It generates a new array with as many items as are in the original array, but in the new array, it puts items in the form that you specify._ |
 
-Open the **`Metals`** module and the **`DiamondSizes`** module. You will see that the list items for metals uses the `for..of` loop that you have seen in several previous projects for building all of the HTML representations of data.
+Open the **`Metals.js`** module and the **`DiamondSizes.js`** module. You will see that the list items for metals uses the `for..of` loop that you have seen in several previous projects for building all of the HTML representations of data.
```
Comment (stevebrownlee) [inline book-4-kneel-diamonds/chapters/KD_RADIO_BUTTONS.md:27]: Please remove the .js extension on the module names

**Review by stevebrownlee — CHANGES_REQUESTED** [review]

File: book-4-kneel-diamonds/chapters/KD_RADIO_BUTTONS.md (hunk 2 above)
Comment (heymonicakay) [reply book-4-kneel-diamonds/chapters/KD_RADIO_BUTTONS.md]: Removed!

File: book-4-kneel-diamonds/chapters/KD_RADIO_BUTTONS.md (hunk 1 above)
Comment (heymonicakay) [reply book-4-kneel-diamonds/chapters/KD_RADIO_BUTTONS.md]: updated!

**Pull request merged**